js/main.js

```diff
@@ -65,3 +92,34 @@ let createTodoElement = (elementId, todoText) => {
   let list = document.getElementById("todoList");
   list.appendChild(todo);
 }
+
+let usernameForm = document.querySelector("#usernameForm")
+
+/* fires when you press enter on the input
+** it gets the name you put in, saves it
+** in localstorage and the change the name
+** on screen
+*/
+let setUsername = event => {
+  event.preventDefault()
+  let newUsernameInput = document.querySelector("#newUsername")
+  if (newUsername.value != "") {
+    storage.setItem("username", newUsername.value)
+    let usernameElement = document.querySelector(".username h1 span")
+    usernameElement.textContent = `${newUsername.value}`
+    newUsernameInput.value = ""
+    usernameForm.setAttribute("hidden", "")
+  } else {
+    usernameForm.setAttribute("hidden", "")
```

Comment:
The else does the same as the if ends

Reply:
Yes, I figured that we want the field to just go away when you submit it empty. So it just hides it again, but skips all the steps to set the user name. It's just in case someone just happens to double-click the name but doesn't wanna change it

Reply:
Okay, that is smart! But it was not what I was referring to, since the lines are the exact same, you can have the line outside the if-statement since no matter what the code will always run `usernameForm.setAttribute("hidden", "")`. 

```
 if (newUsername.value !== "") {
    storage.setItem("username", newUsername.value)
    let usernameElement = document.querySelector(".username h1 span")
    usernameElement.textContent = `${newUsername.value}`
    newUsernameInput.value = ""
  }
usernameForm.setAttribute("hidden", "")
```

This would result exactly the same right?

Reply:
Yeah you're right haha they do the same thing :+1: